modules/displaying-the-insights-status-in-the-web-console.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+[id="displaying-the-insights-status-in-the-web-console.adoc_{context}"]
```

Comment:
@mmuehlfeldRH One thing I missed is that there is a .adoc in the ID, which must be removed. Thanks!

Reply:
Good catch. This was a copy/paste issue. :-)
It's fixed now.